packages/graphql/src/translate/where/create-where-and-params.ts

```diff
@@ -50,7 +50,7 @@ function createWhereAndParams({
         return ["", {}];
     }
 
-    function reducer(res: Res, [key, value]: [string, GraphQLWhereArg]): Res {
+    function reducer(res: Res, [key, value]: [string, any | GraphQLWhereArg | GraphQLWhereArg[]]): Res {
```

Comment:
**question:** what's up with loosening the type safety here with `any`?

Reply:
Not sure if it's loosening -- it's just making the definition explicit for the top level 

This is the def in the types file 
```typescript

export interface GraphQLWhereArg {
    [k: string]: any | GraphQLWhereArg | GraphQLWhereArg[];
    AND?: GraphQLWhereArg[];
    OR?: GraphQLWhereArg[];
}

```

Without making it explicit I was getting ts errors.